packages/driver/src/cy/commands/actions/selectFile.ts

```diff
@@ -259,37 +261,48 @@ export default (Commands, Cypress, cy, state, config) => {
         })
       }
 
-      let eventTarget = subject
+      const getEventTargetFromSubject = (subject) => {
```

Comment:
selectFile did not properly retry its assertions about the subject; eg, it did not wait for the input to become enabled, but threw an error immediately.

I've moved these verifications inside the `subjectFn` we pass to $actionability.verify(), so they'll now properly retry in all situations (not just detached DOM situations).